### PR TITLE
Refs #25170 - Add migration for new seed param naming.

### PR DESCRIPTION
**Pull request by chris1984**

### CODE REVIEW

File: config/foreman.migrations/20181015123542_migrate_seed_params.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+mod = answers['foreman']
+if mod.is_a?(Hash)
+    ['admin_username', 'admin_password', 'admin_first_name', 'admin_last_name', 'admin_email'].each do |var|
+      mod["initial_#{var}"] = mod[var]
```
Comment (ekohl) [inline config/foreman.migrations/20181015123542_migrate_seed_params.rb]: Looks like this needs a `unless mod[var].nil?`

**Review by ekohl — CHANGES_REQUESTED** [review]
Note: you can test this easily by changing `config/foreman-answers.yaml` and add `admin_username: myadmin` under `foreman:`, run `bundle exec rake spec`. It should complain that there's a `initial_admin_username` answer present which wasn't in the answers. It's how I usually test migrations. I'm thinking about making that easier so you can provide some fixtures.

**Comment by chris1984** [comment]
@ekohl this is what I get when I do that, does the output look correct for the manual test?

https://paste.fedoraproject.org/paste/fFVtzNcOKF4tdjUeWHl5iw

**Review by ekohl — CHANGES_REQUESTED** [review]
This now needs to be copied into the katello migration as well.

**Review by ekohl — APPROVED** [review]

**Pull request merged**